src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const MEDIUM_TERM = 3;
+  const MEDIUM_TERM_DISCOUNT = 20;
+
+  let result = PRICE_PER_DAY * days;
+
+  if (days >= LONG_TERM) {
+    result -= LONG_TERM_DISCOUNT;
+  } else if (days >= MEDIUM_TERM) {
+    result -= MEDIUM_TERM_DISCOUNT;
```

Comment:
The discount logic is correct, but it could be improved for clarity. Currently, if `days` is greater than or equal to `LONG_TERM`, only the `LONG_TERM_DISCOUNT` is applied. If `days` is between `MEDIUM_TERM` and `LONG_TERM`, only the `MEDIUM_TERM_DISCOUNT` is applied. Ensure this is the intended behavior, as it might be more intuitive to apply cumulative discounts or different logic if needed.